frontend/ui/isolanguage.lua

```diff
@@ -0,0 +1,530 @@
+    wal = "Wolaitta",
+    war = "Waray",
+    was = "Washo",
+    wen = "Sorbian languages",
+    xal = "Kalmyk",
+    yao = "Yao",
+    yap = "Yapese",
+    ypk = "Yupik languages",
+    zap = "Zapotec",
+    zbl = "Blissymbols",
+    zen = "Zenaga",
+    zgh = "Standard Moroccan Tamazight",
+    znd = "Zande languages",
+    zun = "Zuni",
+    zxx = "No linguistic content",
+    zza = "Zaza",
+
+    -- and some even more specific ones
+    apc = "South Levantine Arabic", -- https://en.wikipedia.org/wiki/South_Levantine_Arabic
+    ary = "Moroccan Arabic",        -- https://en.wikipedia.org/wiki/Moroccan_Arabic
+    cim = "Cimbrian",               -- https://en.wikipedia.org/wiki/Cimbrian_language
+    cmn = "Mandarin",               -- https://en.wikipedia.org/wiki/Mandarin_Chinese
+    dlm = "Dalmatian",              -- https://en.wikipedia.org/wiki/Dalmatian_language
+    hrx = "Hunsrik",                -- https://en.wikipedia.org/wiki/Hunsrik
+    hbs = "Serbo-Croatian",         -- https://en.wikipedia.org/wiki/Serbo-Croatian
+    izh = "Ingrian",                -- https://en.wikipedia.org/wiki/Ingrian_language
+    jje = "Jeju",                   -- https://en.wikipedia.org/wiki/Jeju_language
+    kea = "Kabuverdianu",           -- https://en.wikipedia.org/wiki/Cape_Verdean_Creole
+    lld = "Ladin",                  -- https://en.wikipedia.org/wiki/Ladin_language
+    lzz = "Laz",                    -- https://en.wikipedia.org/wiki/Laz_language
+    mch = "Maquiritari",            -- https://en.wikipedia.org/wiki/Ye%27kuana_language
+    nrf = "Norman",                 -- https://en.wikipedia.org/wiki/Norman_language
+    pdc = "Pennsylvania German",    -- https://en.wikipedia.org/wiki/Pennsylvania_Dutch_language
+    pdt = "Plautdietsch",           -- https://en.wikipedia.org/wiki/Plautdietsch_language
+    phl = "Phalura",                -- https://en.wikipedia.org/wiki/Palula_language
+    stq = "Saterland Frisian",      -- https://en.wikipedia.org/wiki/Saterland_Frisian_language
+    tft = "Ternate",                -- https://en.wikipedia.org/wiki/Ternate_language
+    txg = "Tangut",                 -- https://en.wikipedia.org/wiki/Tangut_language
+    urj = "Uralic Languages",       -- https://en.wikipedia.org/wiki/Uralic_languages
+    vec = "Venetian",               -- https://en.wikipedia.org/wiki/Venetian_language
+    vep = "Veps",                   -- https://en.wikipedia.org/wiki/Veps_language
+    wym = "Vilamovian",             -- https://en.wikipedia.org/wiki/Wymysorys_language
+    yol = "Yola",                   -- https://en.wikipedia.org/wiki/Yola_language
+    yue = "Cantonese",              -- https://en.wikipedia.org/wiki/Cantonese
+    xcl = "Classical Armenian",     -- https://en.wikipedia.org/wiki/Classical_Armenian
+}
+
+local IsoLanguage = {}
+
+function IsoLanguage:getLocalizedLanguage(lang_locale)
+    local language = locale_lang_map[lang_locale]
+    return language and _(language) or lang_locale
```

Comment:
This won't propose any of your language string to translators (and I don't think we want to propose them :) It will just benefit from the ones that may be translater from elsewhere I guess.
So, maybe just don't localize them ?
@Frenzie ?

Reply:
It wouldn't make too much sense to preemptively localize (particularly the likes of Blissymbols), but note that this won't localize much as written, except insofar as the same string is coincidentally localized elsewhere already. :-) The localization would need to happen in isolanguage.lua.

Concretely, only strings that look like _("") or _([[]]) are picked up for localization.